README.md

```diff
@@ -1,2 +1,29 @@
-# Portfolio-setup-and-mobile-version-skeleton
-My Protfolio
+# Project Name
+
+Portfolio-setup-and-mobile-version-skeleton
+
+# PR
```

Comment:
- [ ] I would suggest giving your project an appropriate name. 'Project Name' is merely a dummy text and so is too generic to be your project name. Perhaps you meant: **'Portfolio Setup and Mobile Version skeleton'** - as you have in your next line? 🤔

- [ ] **PR** here is not very descriptive of what the section is all about. Please give the full meaning of **PR**